src/lib/string_sign/tests/test_string_sign.ml

```diff
@@ -142,6 +143,36 @@ let test_other_network_failures () =
     "Other network signature fails with different other network" false
     (verify ~signature_kind:(Other_network "Bar") signature keypair.public_key s)
 
+let test_secret_key_between_scalar_field_and_base_field () =
+  (* There are 86663725065984043395317760 values between the two moduli.
+     - Base:
+       28948022309329048855892746252171976963363056481941560715954676764349967630337
+     - Scalar:
+       28948022309329048855892746252171976963363056481941647379679742748393362948097
+  *)
+  let base_modulus =
+    Bignum_bigint.of_string
+      "28948022309329048855892746252171976963363056481941560715954676764349967630337"
```

Comment:
Is this randomly chosen, or defined somewhere else? 
If first case, I'd use PBT; If second, I'd make this a reference to that definition.

Reply:
> Is this randomly chosen, or defined somewhere else?

No. As the variable name mentions, it is the modulus of the base field of Pallas, the curve used for signing.